package.json

```diff
@@ -29,7 +29,7 @@
   "homepage": "https://github.com/Enterprise-CMCS/macpro-mdct-seds#readme",
   "devDependencies": {
     "@types/dotenv": "^8.2.0",
-    "@types/node": "^14.14.6",
+    "@types/node": "^18.17.0",
```

Comment:
does this matter? i just assumed this was related

Reply:
I believe you're right.